src/Discord.Net.WebSocket/API/Gateway/RequestMembersParams.cs

```diff
@@ -10,8 +10,11 @@ internal class RequestMembersParams
         public string Query { get; set; }
         [JsonProperty("limit")]
         public int Limit { get; set; }
-
         [JsonProperty("guild_id")]
-        public IEnumerable<ulong> GuildIds { get; set; }
+        public ulong GuildId { get; set; }
+        [JsonProperty("user_ids")]
+        public IEnumerable<ulong> UserIds { get; set; }
+        [JsonProperty("nonce")]
+        public string Nonce { get; set; }
```

Comment:
According to [discord docs](https://discord.com/developers/docs/topics/gateway-events#request-guild-members-request-guild-members-structure) `query`, `user_ids` and `string` are optional parameters.
Consider wrapping them into `Optional<>`

Additionally, the `presences` field seems to be also missing & should be also added

Reply:
done